plotly/utils.py

```diff
@@ -35,15 +35,15 @@
 
 http_msg = (
     "The plotly_domain and plotly_api_domain of your config file must start "
-    "with 'https', 'http'.\n If you are not using On-Prem then run the "
+    "with 'https', not 'http'.\n If you are not using On-Premise then run the "
```

Comment:
I believe the `\n If` ... spacing still needs to be fixed. I think it would look cleaner without a new line actually.